ideas.txt

```diff
@@ -0,0 +1,52 @@
+options = arguments in discord slash command
```

Comment:
It's good that you have collected ideas. We usually don't track this as code. Create issues on github to track these ideas.

Reply:
Will do!